app/src/main/java/mozilla/lockbox/view/ItemDetailFragment.kt

```diff
@@ -117,9 +118,10 @@ class ItemDetailFragment : BackableFragment(), ItemDetailView {
 var EditText.readOnly: Boolean
     get() = this.isFocusable
     set(readOnly) {
-        this.isFocusable = !readOnly
+        this.isFocusable = true
         this.isFocusableInTouchMode = !readOnly
-        this.isClickable = !readOnly
+        this.inputType = InputType.TYPE_NULL
+        this.isClickable = true
```

Comment:
Now you're making specializations specific to the detail screen, you should probably put these into their own method. Still use this property, but forcing values for `inputType` `isFocusable` and `isClickable` should go after the call to `readOnly`.